provisioning_templates/snippet/saltstack_setup.erb

```diff
@@ -23,10 +23,12 @@ else
 fi
 <% end -%>
 
-cat > <%= etc_path %>/minion << EOF
+cat > <%= etc_path %>/minion.d/minion.conf << EOF
```

Comment:
Is this path automatically used by salt or do we need to adapt something else?

Reply:
https://docs.saltstack.com/en/latest/ref/configuration/examples.html#example-minion-configuration-file ->

# Per default the minion will automatically include all config files
# from minion.d/*.conf (minion.d is a directory in the same directory
# as the main minion config file).

Reply:
Correct, all files within the .d directory are automatically included. The advantage is that in case the OS package for salt-minion provides a new config file it can update it without any conflicts.